### PR TITLE
Blog post: Guide to free resources for cloud engineering

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/free-resources-for-cloud-engineering/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Innovation is at the heart of cloud engineering. Finding the right cloud solution requires experimentation to understand each cloud ecosystem and the best practices for each. The good news is that cloud providers have free service tiers for many cloud resources. These resources are often always free under a certain usage threshold or free to use for a limited time, often 12 months after creating an account.
+
+Having free access to resources frees you to experiment, try out new ideas, and implement different application deployment architectures. However, given the number of cloud providers and the range of services they provide, getting started can be intimidating. This article is a guide to popular cloud resources and how to use them.
+
+<!--more-->
+
+## AWS
+
+Amazon offers a free tier composed of always free, 12 months free available to new customers, and short term trials. The following is a selection of always free resources with accompanying tutorials, examples, and blog posts to get you started.
```
Comment (stack72) [inline content/blog/free-resources-for-cloud-engineering/index.md:23]: I think we should link to their entire set of free tier docs - https://aws.amazon.com/free/

**Review by stack72 — COMMENTED** [review]
It feels like this post could go out of date easily on each of the individual free tier line items - would it be better to just link to the upstream docs on the line items?

File: content/blog/free-resources-for-cloud-engineering/index.md (hunk 1 above)
Comment (spara) [reply content/blog/free-resources-for-cloud-engineering/index.md:23]: Good suggestion, I'll update this for each provider, it's kinda there for Azure but not obvious

**Comment by spara** [comment]
pulled for reframing

**Pull request closed without merging**